tests/benchmark/parser/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
I'm not sure whether it is appropriate to add these benchmark scripts to the master branch. Why do you think they are necessary?

Reply:
This is not necessary to merge into the master branch. I'm just curious about the actual performance of `Common::Atof` and `fast_double_parser`.

And maybe this benchmark can be helpful for people who want to improve the performance of text parser.

I can remove this commit if you decide to not include it.

Reply:
The benchmark commit is now reverted. It's better to create microbenchmarks for this.